frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/AppearanceSettings.tsx

```diff
@@ -0,0 +1,245 @@
+import { useSelector } from "metabase/lib/redux";
+import { getDocsUrl, getSetting } from "metabase/selectors/settings";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { PLUGIN_SELECTORS } from "metabase/plugins";
+import Select from "metabase/core/components/Select";
+import { useUniqueId } from "metabase/hooks/use-unique-id";
+import { color } from "metabase/lib/colors";
+
+import type {
+  ActivePreviewPane,
+  EmbeddingDisplayOptions,
+  EmbeddingParameters,
+  EmbedResource,
+  EmbedResourceType,
+  EmbedType,
+} from "../EmbedModal.types";
+import EmbedCodePane from "./EmbedCodePane";
+import PreviewPane from "./PreviewPane";
+import {
+  CODE_PREVIEW_CONTROL_OPTIONS,
+  DisplayOptionSection,
+  SettingsTabLayout,
+} from "./StaticEmbedSetupPane.styled";
+import { StaticEmbedSetupPaneSettingsContentSection } from "./StaticEmbedSetupPaneSettingsContentSection";
+
+const THEME_OPTIONS = [
+  { label: t`Light`, value: "light" },
+  { label: t`Dark`, value: "night" },
+  { label: t`Transparent`, value: "transparent" },
+];
+const DEFAULT_THEME = THEME_OPTIONS[0].value;
+
+export interface AppearanceSettingsProps {
+  activePane: ActivePreviewPane;
+
+  embedType: EmbedType;
+  resource: EmbedResource;
+  resourceType: EmbedResourceType;
+  iframeUrl: string;
+  token: string;
+  siteUrl: string;
+  secretKey: string;
+  params: EmbeddingParameters;
+  displayOptions: EmbeddingDisplayOptions;
+
+  onChangePane: (pane: ActivePreviewPane) => void;
+  onChangeDisplayOptions: (displayOptions: EmbeddingDisplayOptions) => void;
+}
+
+export const AppearanceSettings = ({
+  activePane,
+  embedType,
+  resource,
+  resourceType,
+  iframeUrl,
+  token,
+  siteUrl,
+  secretKey,
+  params,
+  displayOptions,
+
+  onChangePane,
+  onChangeDisplayOptions,
+}: AppearanceSettingsProps): JSX.Element => {
+  const docsUrl = useSelector(state =>
+    getDocsUrl(state, { page: "embedding/static-embedding" }),
+  );
+  const canWhitelabel = useSelector(PLUGIN_SELECTORS.canWhitelabel);
+  const availableFonts = useSelector(state =>
+    getSetting(state, "available-fonts"),
+  );
+
+  const fontControlLabelId = useUniqueId("display-option");
+
+  return (
+    <SettingsTabLayout
+      settingsSlot={
+        <>
+          <StaticEmbedSetupPaneSettingsContentSection
+            title={t`Customizing your embed’s appearance`}
+          >
+            <Text>{jt`These cosmetic options requiring changing the server code. You can play around with and preview the options here, and check out the ${(
+              <ExternalLink
+                key="doc"
+                href={docsUrl}
+              >{t`documentation`}</ExternalLink>
+            )} for more.`}</Text>
+          </StaticEmbedSetupPaneSettingsContentSection>
+          <StaticEmbedSetupPaneSettingsContentSection
+            title={t`Play with the options here`}
+            mt="2rem"
+          >
+            <Stack spacing="1rem">
```

Comment:
I'm usually really biased to splitting these children into subcomponents - i.e. `FontControl` or `BackgroundControl` or things like that. I think it would make the structure a lot easier to look at and help us test/separate concerns more easily. What do you think?